ProcessLib/LIE/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -93,8 +95,30 @@ HydroMechanicsProcess<GlobalDim>::HydroMechanicsProcess(
                 new BoundaryConditionBuilder(
                     *_process_data.fracture_property.get())));
     }
+
+    if (!_process_data.deactivate_matrix_in_flow)
+    {
+        _process_data.p_element_status.reset(new MeshLib::ElementStatus(&mesh));
+    }
+    else
```

Comment:
since it's not sure if `reset()` is always the final task (not necessary) in the constructor, I would keep current.